Library/Formula/ndjbdns.rb

```diff
@@ -0,0 +1,106 @@
+class Ndjbdns < Formula
+  desc "N-DJBDNS is a brand new release of DJBDNS."
```

Comment:
It's their lead line from http://pjp.dgplug.org/ndjbdns/, I've added fork in there.